app/controllers/users_controller.rb

```diff
@@ -1,35 +1,36 @@
+# frozen_string_literal: true
```

Comment:
Just curious - what purpose does this serve???

Reply:
Looks like it's on a lot of files, but not every file. What's the criteria for adding this to a file?

Reply:
@sanghun89 It the equivalent of putting "mystring".freeze on every string in the file. It makes them all immutable, kinda like a const in JS. It also saves about 20% on the garbage collection and is going to be default on in Ruby 3.0. Kinda minor but it's recommended for performance and safety. An @markdamobi it'll get automatically put on any run time ruby file when rubocop runs.